api/src/main/java/org/pmiops/workbench/compliancetraining/ComplianceTrainingServiceImpl.java

```diff
@@ -81,6 +81,8 @@ public DbUser syncComplianceTrainingStatus()
       throws org.pmiops.workbench.moodle.ApiException, NotFoundException, ApiException {
     DbUser user = userProvider.get();
 
+    log.info(String.format("Syncing compliance training status for user %s", user.getUsername()));
```

Comment:
We log `<user> logged in` but it's difficult to sub-search for a specific request trace. This line allows us to quickly find relevant request traces for compliance training sync errors.